Drop-In/src/main/java/com/braintreepayments/api/DropInActivity.java

```diff
@@ -261,7 +253,7 @@ private void onDidShowBottomSheet() {
     }
 
     private void onDidHideBottomSheet() {
-        finishDropInWithPendingResult(true);
+        finishDropInWithPendingResult(false);
```

Comment:
I think this needs to be true - when it's false I see that weird UI bug that looks like the toolbar is sliding down with drop-in. If it does get set back to true, it looks like it's true in all usages - do we still need this true/false parameter at all?

Reply:
Oh right yeah there we may have to add an additional piece of logic to force a fade-out animation. I have an idea of how to make it more expressive.

Reply:
Ended up adding an enum. The default system animation for an activity exit might cause the toolbar bug to occur. If we explicitly declare a `fade_out` that should resolve the issue.